docker/datomic/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM demonsthere/orcpub:bootstrap-lein
```

Comment:
There are datomic images, which I think we should use.
See: https://github.com/alexanderkiel/datomic-free